scripts/opensearch-build/release.jenkinsfile

```diff
@@ -0,0 +1,104 @@
+properties([
+  parameters([
```

Comment:
The 'minimum' addressable unit for a release seems to be a distribution manifest.  However this might not be a one-size-fits -all solution so I wanted to be sure that parameters could be added for other project types.

I believe its useful to see the parameter defined here as a property so its clear within the `standardReleasePipeline {...}` body code of this file that it can be accessed via `params.DistManifestUrl`